vuepress/docs/next/tutorials/devops/plugin-environment-variables.md

```diff
@@ -2,60 +2,54 @@
+The `entando.json` file makes environment variables available to a microservice in a docker-based bundle. There are two options:
 1. Inject the variables using a key-value pair
 2. Inject a reference to an existing Kubernetes Secret 
 
-This tutorial will demonstrate both of these options. The Entando Plugin will receive one environment value directly as plain text in the Pod YAML and the other indirectly via a referenced Secret.
+This tutorial will demonstrate both of these options. The microservice will receive one environment value directly as plain text in the pod YAML and the other indirectly through a referenced Secret.
 
 ## Prerequisites
-* [A project bundle](../create/ms/generate-microservices-and-micro-frontends.md) using the latest Entando Blueprint. 
+* [A bundle project using the latest Entando Blueprint](../create/ms/generate-microservices-and-micro-frontends.md) 
 
-## Add Environment Variables to the Plugin Descriptor File
-1. Determine YOUR-BUNDLE-ID. You can do this from your project directory using this command:
-```sh
-ent prj get-bundle-id
-```
-Alternatively, supply the full bundle URL (including the .git suffix):
+## Add Environment Variables to the Microservice
+1. To determine YOUR-BUNDLE-ID, run the following command from the root project directory. Supply the full bundle URL (including the .git suffix).
 ```sh
 ent ecr get-bundle-id https://github.com/YOUR-ACCOUNT/YOUR-PLUGIN-BUNDLE.git
 ```
-For example, here's the bundle ID for the first [Standard Banking Demo bundle](../solution/install-standard-demo.md):
-```
-$ent ecr https://github.com/entando-samples/standard-demo-banking-bundle.git
-343826ca
-```
 
-2. Now create a Secret named `YOUR-BUNDLE-ID-my-secret` with a key-value pair `mySecretKey=mySecretValue`. Make sure to replace YOUR-BUNDLE-ID with the value from the previous step. 
+2. Create a Secret named `YOUR-BUNDLE-ID-my-secret` with key-value pair `mySecretKey=mySecretValue`. Replace YOUR-BUNDLE-ID with the output of the previous step. 
 ```sh
 kubectl create secret generic YOUR-BUNDLE-ID-my-secret --from-literal=mySecretKey=mySecretValue -n entando
 ```
-3. Verify that the plugin file `bundle/plugins/YOUR-PLUGIN.yaml` specifies `descriptorVersion: v4` or add it if you're upgrading an existing bundle.
-4. Insert the following `environmentVariables` section into `YOUR-PLUGIN.yaml` after replacing `YOUR-BUNDLE-ID` with the correct value. By convention, environment variables are all caps and K8s resource names are hyphenated.
-```yaml
-environmentVariables:
-  - name: SIMPLE_VAR
-    value: mySimpleValue
-  - name: SECRET_VAR
-    valueFrom:
-      secretKeyRef:
-        name: YOUR-BUNDLE-ID-my-secret
-        key: mySecretKey
+
+3. Insert the following `env` section into the microservice in `entando.json`, remembering to replace `YOUR-BUNDLE-ID`. By convention, environment variables are all caps and K8s resource names are hyphenated.
```

Comment:
add the or your >> the entando.json, ...

Reply:
not needed

Reply:
add something to the bag, add an apple into the recipe, insert something into the iphone, >> thinking they all need an article. Not completely sure but entando.json is like a toyota, there can be many different ones and you wouldn't say put something into toyota??

Reply:
<img width="731" alt="Screen Shot 2022-10-04 at 10 15 56 AM" src="https://user-images.githubusercontent.com/93157274/193884133-94e37f32-3c53-441f-a402-e38f9878ab83.png">

Reply:
https://english.stackexchange.com/questions/399258/should-i-use-the-before-file-names